### PR TITLE
Prepare for final 1.0.0 release

### PR DESCRIPTION
**Pull request by ssayer**
Preparing to publish 1.0.0 version of this as a gem in preparation for Bonnie 2.0 release. Also standardize authors.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [ ] JIRA ticket for this PR:
- [ ] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered.
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here:
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here:
- [ ] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA test links:
- [ ] Justification for using JIRA tests:
- [ ] JIRA tests have been added to sprint

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases NA
- [x] You have tried to break the code NA

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass NA
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree NA

### CODE REVIEW

File: cql_qdm_patientapi.gemspec [hunk 1]
```diff
@@ -6,7 +6,7 @@ require 'cql_qdm_patientapi/version'
 Gem::Specification.new do |spec|
   spec.name          = "cql_qdm_patientapi"
   spec.version       = CqlQdmPatientapi::VERSION
-  spec.authors     = ['Adam M. Holmes', 'James Bradley']
+  spec.authors     = ['The MITRE Corporation']
   spec.email       = ['aholmes@mitre.org', 'jhbradley@mitre.org']
```
Comment (losborne) [inline cql_qdm_patientapi.gemspec]: Should these emails be changed as well?

**Comment by ssayer** [comment]
Good point... changed.
